src/lib_ccx/general_loop.c

```diff
@@ -677,6 +677,10 @@ int process_data(struct encoder_ctx *enc_ctx, struct lib_cc_decode *dec_ctx, str
 			if (ret == CCX_EINVAL)
 				return ret;
 		}
+		else {
```

Comment:
generally in code, if you check else statement is written like
```
if(condition)
{
    //statement
}
else
{
    \\statement
}
```
we do not do use  K&R coding style in this file
```
if (condition) {
    //statement
} else {
    //statement
}
```